js/functions.js

```diff
@@ -0,0 +1,71 @@
+function checkLength1 (row, maximalLength) {
+  return row.length >= maximalLength ? 'false' : 'true';
+}
+
+console.log(checkLength1('hello my name is kate', 2));
+
+function checkLength2 (row, maximalLength) {
+  if (row.length <= maximalLength) {
+    return true;
+  }
+  return false;
+}
+
+console.log(checkLength2 ('hi whatsup', 3))
+
+function isPalindrome (word) {
+  const newWord = word.split(' ');
+  let newString = '';
+  for (let i = 0; i < newWord.length; i++) {
+    newWord[i] = newWord[i].toLowerCase();
+    newString += newWord[i];
+  }
+
+  for (let i = 0; i < newString.length; i++) {
+    if (newString[i] === newString.at(-(i + 1))) {
+      return true;
+    }
+    return false;
+  }
+}
+console.log(isPalindrome('holo9holo'));
+
+function getNumber (row) {
+  if (typeof row === 'number') {
+    row = String(row);
+  }
+
+  let newNumber = '';
+
+  for (let i = 0; i < row.length; i++) {
+    if (Number(row[i]) <= 9 && Number(row[i] > 0)) {
```

Comment:
```suggestion
    if (row[i] !== ' ' && !Number.isNaN(+row[i])) {
```

Так будет читаться чуть проще))
- Если
- элемент строки не является пробелом // `row[i] !== ' '`
- и
- приведенный к числу элемент строки // `+row[i]`
- не является "not a number" (NaN) // `!Number.isNaN()`